packages/mattermost-redux/src/reducers/entities/threads/counts.ts

```diff
@@ -142,8 +147,26 @@ export function countsIncludingDirectReducer(state: ThreadsState['counts'] = {},
     switch (action.type) {
     case ThreadTypes.ALL_TEAM_THREADS_READ:
         return handleAllTeamThreadsRead(state, action);
-    case ThreadTypes.READ_CHANGED_THREAD:
-        return handleReadChangedThread(state, action);
+    case ThreadTypes.READ_CHANGED_THREAD: {
+        const {teamId, channelType} = action.data;
+        if (teamId === '' || isDmGmChannel(channelType)) {
```

Comment:
Do we still need to check the channel type here if we're passing in the channel's team ID? It seems like we should only need to check one or the other

Reply:
@hmhealey sometimes the server sends a team id along dm/gm threads, I was reluctant at first to change the server, but I'll try to do so